ext/cl_khr_extended_versioning.asciidoc

```diff
@@ -0,0 +1,284 @@
+// Copyright 2019 The Khronos Group. This work is licensed under a
+// Creative Commons Attribution 4.0 International License; see
+// http://creativecommons.org/licenses/by/4.0/
+
+[[cl_khr_extended_versioning]]
+== Extended versioning
+
+This extension introduces new platform and device queries that return detailed
+version information to applications. It makes it possible to return the exact
+revision of the specification or intermediate languages supported by an
+implementation. It also enables implementations to communicate a version
+number for each of the extensions they support and remove the requirement
+for applications to process strings to test for the presence of an extension or
+intermediate language or built-in kernel.
+
+=== General information
+
+==== Name Strings
+
+`cl_khr_extended_versioning`
+
+==== Contributors
```

Comment:
See previous comment regarding extension metadata like contributors, issues, etc.  We haven't included this information in other KHR extensions.  Is this a practice we want to continue into the future?

Reply:
I think my position hasn't changed here and I think we should do that for all extensions.